### ISSUE TITLE
Make issue

### ISSUE THREAD

**Issue by asanchez500k** [lucasjones/cpuminer-multi#97]
Im trying to compile on a PS3 running Debian 8 Jessie. Autogen.sh and configure.sh work perfect but make gets an error at the end of the compilation about sysinfos.c . Apparently assembly cannot find the cpuid or something. I do not know how to fix this but I can't wait to mine some how.... Here is the log file... 

cd . && /bin/bash ./config.status Makefile depfiles
config.status: creating Makefile
config.status: executing depfiles commands
cd . && /bin/bash ./config.status cpuminer-config.h
config.status: creating cpuminer-config.h
config.status: cpuminer-config.h is unchanged
make  all-recursive
make[1]: Entering directory `/home/asanchez500/Desktop/cpuminer-multi-linux'
Making all in compat
make[2]: Entering directory `/home/asanchez500/Desktop/cpuminer-multi-linux/compat'
 cd .. && /bin/bash ./config.status compat/Makefile 
config.status: creating compat/Makefile
make[2]: Leaving directory `/home/asanchez500/Desktop/cpuminer-multi-linux/compat'
make[2]: Entering directory `/home/asanchez500/Desktop/cpuminer-multi-linux/compat'
make[3]: Entering directory `/home/asanchez500/Desktop/cpuminer-multi-linux/compat'
make[3]: Nothing to be done for `all-am'.
make[3]: Leaving directory `/home/asanchez500/Desktop/cpuminer-multi-linux/compat'
make[2]: Leaving directory `/home/asanchez500/Desktop/cpuminer-multi-linux/compat'
make[2]: Entering directory `/home/asanchez500/Desktop/cpuminer-multi-linux'
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -fno-strict-aliasing  -I.  -Wno-pointer-sign -Wno-pointer-to-int-cast   -g -O2  -MT cpuminer-sysinfos.o -MD -MP -MF .deps/cpuminer-sysinfos.Tpo -c -o cpuminer-sysinfos.o `test -f 'sysinfos.c' || echo './'`sysinfos.c
sysinfos.c: In function ‘has_aes_ni’:
sysinfos.c:122:2: error: impossible register constraint in ‘asm’
  asm volatile("cpuid" : "=a"(a), "=b"(b), "=c"(c), "=d"(d) : "a"(functionnumber), "c"(0));
  ^
make[2]: *** [cpuminer-sysinfos.o] Error 1
make[2]: Leaving directory `/home/asanchez500/Desktop/cpuminer-multi-linux'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/asanchez500/Desktop/cpuminer-multi-linux'
make: *** [all] Error 2

**Comment by DarrenRainey** [comment]
By PS3 I am assuming you mean PlayStation 3. The problem is  that the miner only supports the x86 and ARM processor architectures where as the PS3 uses a PowerPC based CPU which isn't compatible without some heavy modifications.

A PS3 is not that efficient so unless you have free power you won't make anything and I wouldn't expect high hash rates either.

**Comment by asanchez500k** [comment]
I was able to get the original cpuminer to mine litecoin and bitcoin. Im surprised this won't compile at all. The hash rate for the PS3 for litecoin is 43k H/s. Which is noting these days due to difficulty but could prove lucrative with Monero. But I can't get any miner to work. I guess I could try virtualizing x86 with QEMU and see if it can mine.... I will let you guys know.

**Comment by DarrenRainey** [comment]
Yeah the reason the reason the old cpuminer worked was because it support the PowerPC architecture whereas this one has dropped support for everything expect x86 and ARM. You could try using a JavaScript miner but again pretty much any modern PC,laptop or even a raspberry pi would probably have a higher hash rate with less power usage.

**Comment by juanpc2018** [comment]
did a small research about old cpu & gpu miners for Bitcoin...
it may help...
https://bitcointalk.org/index.php?topic=2592063.msg26388321
